cropper/src/main/res/drawable/ic_crop_shape_square_24.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="@android:color/white"
```

Comment:
This has issue performance in APIs under 21.
Please change to `android:fillColor="#ffffffff"`